gateway/mw_persist_graphql_operation.go

```diff
@@ -0,0 +1,93 @@
+package gateway
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"io"
+	"net/http"
+	"strings"
+
+	"github.com/TykTechnologies/tyk/apidef"
+)
+
+// PersistGraphQLOperationMiddleware lets you convert any HTTP request into a GraphQL Operation
+type PersistGraphQLOperationMiddleware struct {
+	BaseMiddleware
+}
+
+func (i *PersistGraphQLOperationMiddleware) Name() string {
+	return "PersistGraphQLOperationMiddleware"
+}
+
+func (i *PersistGraphQLOperationMiddleware) EnabledForSpec() bool {
+	return true
```

Comment:
This always returns true which means the persistent query feature is always enabled, right? Do we need to add a new config parameter to the API def?

Reply:
Oh no it's always enabled, the version and extended path are what determine if it's going to be used on a request.